compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_globals.rs

```diff
@@ -195,26 +265,43 @@ impl BrilligGlobals {
             for map in entry_point_allocations {
                 globals_allocations.extend(map);
             }
+
+            // The global memory space is not shared across entry points.
+            // Thus, if a call can be used by two entry points we should not use the hoisted constant allocations.
+            if entry_points.len() == 1 {
+                let hoisted_allocations = self.entry_point_hoisted_globals_map.get(&entry_points[0]).unwrap_or_else(|| panic!("ICE: Expected hoisted constant allocations to be set for function {brillig_function_id}"));
```

Comment:
```suggestion
                let hoisted_allocations = self
                    .entry_point_hoisted_globals_map
                    .get(&entry_points[0])
                    .unwrap_or_else(|| panic!("ICE: Expected hoisted constant allocations to be set for function {brillig_function_id}"));
```

Reply:
This is what fmt does. I could try and split up the `get` and the `unwrap_or_else` if we would like it split on multiple lines.

Reply:
Sure. I'm surprised that `fmt` makes a 200+ char line.

Reply:
Yeah I was surprised by it as well. I split up the `get` and `unwrap_or_else`